recipes-asteroid/asteroid-settings/asteroid-settings_git.bb

```diff
@@ -11,4 +11,11 @@ S = "${WORKDIR}/git"
 inherit qmake5
 
 DEPENDS += "qml-asteroid nemo-qml-plugin-systemsettings nemo-qml-plugin-dbus qtconnectivity mapplauncherd-booster-qtcomponents"
-RDEPENDS_${PN} += "qtconnectivity-qmlplugins nemo-qml-plugin-systemsettings nemo-qml-plugin-dbus"
+RDEPENDS_${PN} += "qtconnectivity-qmlplugins nemo-qml-plugin-systemsettings nemo-qml-plugin-dbus qtquickcontrols-qmlplugins"
+FILES_${PN} += "/usr/share/translations/"
+
+do_install_append() {
+    lrelease ${WORKDIR}/git/asteroid-settings.pro
+    install -d ${D}/usr/share/translations/
+    cp ${WORKDIR}/git/asteroid-settings.*.qm ${D}/usr/share/translations/
```

Comment:
Languages aren't picked up by the normal Qt compiler, we specifically need to run lrelease to create the "compiled language files". If you mean the install -d and cp part, I don't think that can be set up in asteroid-settings.pro. Which files to compile from .ts to .qm however, is defined in asteroid-settings.pro. See https://github.com/TheLastProject/asteroid-settings/blob/8066edab1197269e8d0e10d15e1251e0482976f4/asteroid-settings.pro#L21-L36 for that.